marabunta/model.py

```diff
@@ -7,7 +7,7 @@
 
 from distutils.version import StrictVersion
 try:  # Python 2.x
-    from cStringIO import StringIO
+    from io import StringIO
 except ImportError:  # Python 3.x
     from io import StringIO
```

Comment:
You try twice to import StringIO here from io lib.

Either you keep it the old way which shoud be compatible in both python version or keep only the import from io

http://python-future.org/compatible_idioms.html#stringio